editor.planx.uk/src/pages/FlowEditor/components/Sidebar/Publish/AlteredExternalPortals.tsx

```diff
@@ -0,0 +1,107 @@
+import Box from "@mui/material/Box";
+import Link from "@mui/material/Link";
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import ListItemText from "@mui/material/ListItemText";
+import Typography from "@mui/material/Typography";
+import { formatLastPublishMessage } from "pages/FlowEditor/utils";
+import React, { useState } from "react";
+import { useAsync } from "react-use";
+import BlockQuote from "ui/editor/BlockQuote";
+
+import { useStore } from "../../../lib/store";
+
+export interface ExternalPortal {
+  text: string;
+  flowId: string;
+  publishedFlowId: number;
+  summary: string;
+  publishedBy: number;
+  publishedAt: string;
+}
+
+const AlteredExternalPortalListItem = (props: ExternalPortal) => {
+  const { text, flowId, publishedFlowId, summary, publishedAt } = props;
+
+  const [externalPortalLastPublishedTitle, setNestedFlowLastPublishedTitle] =
+    useState<string>();
+  const lastPublisher = useStore((state) => state.lastPublisher);
+
+  const _externalPortalLastPublishedRequest = useAsync(async () => {
+    const user = await lastPublisher(flowId);
+    setNestedFlowLastPublishedTitle(
+      formatLastPublishMessage(publishedAt, user),
+    );
+  });
+
+  return (
+    <ListItem
+      key={publishedFlowId}
+      disablePadding
+      sx={{
+        backgroundColor: "border.input",
+        color: "common.white",
+        px: 1.5,
+        py: 0.7,
+        margin: 0,
+        mb: 1,
+      }}
+    >
+      <ListItemText
+        sx={{ color: "common.white" }}
```

Comment:
We need to handle the focus style here also, currently we get black text on a black background.


https://github.com/user-attachments/assets/71d64a9f-f9fa-4062-bdd6-c8a9ebde6909